WordPress/Classes/Utility/WPUserAgent.h

```diff
@@ -6,25 +6,16 @@
  */
 @interface WPUserAgent : NSObject
 
-#pragma mark - Changing the user agent
-
 /**
- *  @brief      Sets the App's user agent to be the default one.
+ *  @brief      Sets User-Agent header of all UIWebViews to be the WordPress one.
  */
-- (void)useDefaultUserAgent;
-
-/**
- *  @brief      Sets the App's user agent to be the WordPress one.
- */
-- (void)useWordPressUserAgent;
-
-#pragma mark - Getting the user agent
+- (void)useWordPressUserAgentInUIWebViews;
 
 /**
- *  @brief      Call this method to get the current user agent.
+ *  @brief      Get WordPress User-Agent header.
  *
- *  @returns    The current user agent.
+ *  @returns    WordPress custom User-Agent header.
  */
-- (NSString *)currentUserAgent;
+- (NSString *)wordPressUserAgent;
```

Comment:
By convention, we'd prefer to define this as a readonly @property instead of a method in the interface file. Then in the implementation we'd redefine it as a readwrite property.